pkg/nuclide/file-tree/spec/FileTreeController-spec.js

```diff
@@ -66,6 +66,21 @@ describe('FileTreeController', () => {
     fileTreeController.destroy();
   });
 
+  it('checks if core:backspace and core:delete is registered on .nuclide-file-tree', () => {
+    var timesCalled = 0;
+    // Find div element
+    var el = fileTreeController._panelController._hostEl.getElementsByClassName('nuclide-file-tree')[0];
```

Comment:
I managed to get this work
```javascript
      var el = React.findDOMNode(TestUtils.findRenderedDOMComponentWithClass(
        fileTreeController._panelController.getChildComponent(),
        'nuclide-file-tree'
      ));
```
as `getTreeComponent` doesn't return `<div className="nuclide-file-tree">`

Is that ok?